subprojects/docs/src/docs/userguide/installation.adoc

```diff
@@ -15,45 +15,148 @@
 [[installation]]
 == Installing Gradle
 
+You can install the Gradle build tool on Linux, macOS, or Windows.
+This document covers installing using a package manager like SDKMAN!, Homebrew, or Scoop, as well as manual installation.
+
+To upgrade Gradle, use of the <<sec:upgrading_wrapper,Gradle Wrapper>> is the recommended.
+
+You can find all releases and their checksums on the link:https://gradle.org/releases[releases page].
 
 [[sec:prerequisites]]
 === Prerequisites
+Gradle runs on all major operating systems and requires only a link:http://www.oracle.com/technetwork/java/javase/downloads/index.html[Java JDK] version 7 or higher to be installed. To check, run `java -version`. You should see something like this:
+
+----
+❯ java -version
+java version "1.8.0_151"
+Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
+Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)
+----
 
-Gradle requires a Java JDK or JRE to be installed, version 7 or higher (to check, use `java -version`). Gradle ships with its own Groovy library, therefore Groovy does not need to be installed. Any existing Groovy installation is ignored by Gradle.
+Gradle ships with its own Groovy library, therefore Groovy does not need to be installed. Any existing Groovy installation is ignored by Gradle.
```

Comment:
If we mention Groovy here then we should also mention Kotlin. BTW: I think we should also render the used Kotlin version in the output of `gradle -version`. More a topic for the Kotlin DSL team.

Reply:
Filed https://github.com/gradle/kotlin-dsl/issues/729